lib/Common/Core/PerfCounter.h

```diff
@@ -150,7 +151,7 @@ namespace PerfCounter
     class RecyclerTrackerCounterSetDefinition
     {
     public:
-        static DWORD const MaxCounter = 46;
+        static DWORD const MaxCounter = 48;
```

Comment:
Is this supposed to be bumped-up by 2 instead of 1? I could believe it but am curious as to why.

Reply:
perhaps I'm missing something, but I'm not seeing where you added new entries?  Also, not sure what you're intending to track here. Did you want this hooked up to telemetry data?

Reply:
I was following what LiteralString was doing.  The new entries are in the ChakraFull PR.  I'm not really intending to track anything here, wasn't sure whether we should be tracking all string types or if I should just remove this scaffolding.